### PR TITLE
MEV cleanup

### PR DESCRIPTION
**Pull request by nkryuchkov**
Adjust comments and docs

### CODE REVIEW

**Comment by nkryuchkov** [comment]
On hold until it's tested

**Comment by nkryuchkov** [comment]
Blocked by https://github.com/bloxapp/ssv/pull/952/commits/4b1b42302bd3504a85db07781e78e0e0bd5c9924

File: beacon/goclient/goclient.go [hunk 1]
```diff
@@ -125,7 +125,11 @@ var _ metrics.HealthCheckAgent = &goClient{}
 
 // New init new client and go-client instance
 func New(logger *zap.Logger, opt beaconprotocol.Options, operatorID spectypes.OperatorID, slotTicker slot_ticker.Ticker) (beaconprotocol.Beacon, error) {
-	logger.Info("consensus client: connecting", fields.Address(opt.BeaconNodeAddr), fields.Network(opt.Network))
+	logger.Info("consensus client: connecting",
```
Comment (nkryuchkov) [inline beacon/goclient/goclient.go:128]: to be reverted

File: beacon/goclient/goclient.go [hunk 2]
```diff
@@ -138,7 +142,11 @@ func New(logger *zap.Logger, opt beaconprotocol.Options, operatorID spectypes.Op
 		return nil, errors.WithMessage(err, "failed to create http client")
 	}
 
-	logger.Info("consensus client: connected", fields.Name(httpClient.Name()), fields.Address(httpClient.Address()))
+	logger.Info("consensus client: connected",
```
Comment (nkryuchkov) [inline beacon/goclient/goclient.go:145]: to be reverted

File: logging/fields/fields.go [hunk 3]
```diff
@@ -298,3 +299,7 @@ func FormatDutyID(epoch phase0.Epoch, duty *spectypes.Duty) string {
 func Root(r [32]byte) zap.Field {
 	return zap.String("root", hex.EncodeToString(r[:]))
 }
+
+func GasLimit(gasLimit uint64) zap.Field {
```
Comment (nkryuchkov) [inline logging/fields/fields.go:303]: to be reverted

File: logging/fields/fields.go [hunk 4]
```diff
@@ -52,6 +52,7 @@ const (
 	FieldFeeRecipient        = "fee_recipient"
 	FieldFork                = "fork"
 	FieldFromBlock           = "from_block"
+	FieldGasLimit            = "gas_limit"
```
Comment (nkryuchkov) [inline logging/fields/fields.go:55]: to be reverted

File: protocol/v2/blockchain/beacon/client.go [hunk 5]
```diff
@@ -64,5 +64,5 @@ type Options struct {
 	MinGenesisTime uint64 `yaml:"MinGenesisTime" env:"MinGenesisTime"`
 	BeaconNodeAddr string `yaml:"BeaconNodeAddr" env:"BEACON_NODE_ADDR" env-required:"true"`
 	Graffiti       []byte
-	GasLimit       uint64 `yaml:"GasLimit" env:"GAS_LIMIT" env-default:"30000000" env-description:"Gas limit used for beacon node actions"`
+	GasLimit       uint64
```
Comment (nkryuchkov) [inline protocol/v2/blockchain/beacon/client.go:67]: To be deleted

File: protocol/v2/ssv/runner/validator_registration.go [hunk 6]
```diff
@@ -22,7 +21,7 @@ import (
 
 type ValidatorRegistrationRunner struct {
 	BaseRunner *BaseRunner
-	GasLimit   uint64
+	GasLimit   uint64 // TODO: spec will use a constant gas limit, so this line needs to be removed, gas limit should be taken from spec
```
Comment (nkryuchkov) [inline protocol/v2/ssv/runner/validator_registration.go:24]: this should be done before merging

**Review by moshe-blox — APPROVED** [review]

**Pull request merged**